packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.spec.ts

```diff
@@ -196,6 +213,28 @@ describe('NodeHttpRequestResponseHandler', () => {
 
     });
 
+    it('should set the charset to utf-8 if no content-type header is specified', async () => {
```

Comment:
To be complete, this also needs to tests the same if such a header is present but misses a charset.

Reply:
Well actually my description is incorrect. This tests that. Since I do pass a content-type header without a charset in this test.
```suggestion
    it('should set the charset to utf-8 if no charset is specified in the content-type header', async () => {
```